src/ctia/bulk/core.clj

```diff
@@ -244,15 +267,28 @@
   The create-entities set the enveloped-result? to True in the flow
   configuration to get :data and :tempids for each entity in the result."
   [entities-by-type]
-  (->> entities-by-type
-       (map (fn [[_ v]] (:tempids v)))
-       (reduce into {})))
+  (into {}
+        (map (fn [[_ v]] (:tempids v)))
+         entities-by-type))
 
 (defn bulk-refresh? [get-in-config]
   (get-in-config
     [:ctia :store :bulk-refresh]
     "false"))
 
+(defn bulk-size [bulk]
+  (reduce + (map count (vals bulk))))
+
+(defn get-bulk-max-size [get-in-config]
+  (get-in-config [:ctia :http :bulk :max-size]))
+
+(defn validate-bulk-size!
+  [bulk
+   {{:keys [get-in-config]} :ConfigService}]
+  (when (> (bulk-size bulk) (get-bulk-max-size get-in-config))
```

Comment:
```suggestion
  (when (< (get-bulk-max-size get-in-config) (bulk-size bulk))
```

Per long lasting tradition, Lispers prefer using `<` and `<=`. "Elements of Clojure" nicely illustrates that in "Idioms" chapter:

> But the most confusing operators, by far, are the inequalities. As children, we were often taught the meaning of 3 > 1 by giving the > tiny teeth, transforming it into a hungry alligator that always wants to eat the largest number. Here the infix notation isn’t an incidental detail; it’s central to our mental model.
> Small wonder, then, that (> a b) is difficult to read. The > doesn’t point at anything; we have to move it over by one term before realizing that the hungry alligator wants to eat a. Here, too, there is a useful mnemonic: we can think of > as a downward slope, and < as an upward slope. This way, we can read (> a b c) as a descending collection of values fromatocand(< a b c)as ascending.
> Even with this mnemonic, readers can get confused. Subtraction and division, at least, have a fixed relationship between the left and right terms. With each inequality, we have to decide anew whether to use (< a b) or (> b a). Almost always, this decision
> is arbitrary; we can reduce the burden on the reader by choosing one and sticking with it.
> Inequalities should be ordered least to greatest, except where reordering terms hurts code clarity.